p2p/config/config.go

```diff
@@ -77,13 +77,13 @@ func DefaultConfig() Config {
 		DialTimeout:           duration("1m"),
 		ConnKeepAlive:         duration("48h"),
 		NetworkID:             TestNet,
-		ResponseTimeout:       duration("15s"),
-		SessionTimeout:        duration("5s"),
+		ResponseTimeout:       duration("60s"),
```

Comment:
You can revert timeouts to 15 s again

Reply:
I think we should revisit these timeouts again anyway. it might take upto 20sec to send some messages with really high throughput. though I'm still testing this to figure out the exact numbers, they're going to be more than 15s anyway.